### DISCUSSION TITLE
Privacy policy

### DISCUSSION THREAD

**Discussion by Oridaro** [Flow-Launcher/Flow.Launcher#1218] [Q&A] [answered]
HI! Where I can found privacy policy?

**Comment by jjw24** [comment, marked as answer]
Hey how's it going?

We don't have a privacy policy because this project is 100% transparent and open source. Everything in this Flow Launcher repository does not send any information externally and we do not collect anything from the user.

Worth noting while currently all plugins outside of this repository that are published in the plugin store have their code inspected prior to acceptance, it is still up to the user to do their due diligence to ensure the plugins they will be installing via Flow Launcher are safe to use and do not intrude on privacy.
